tests-resources/projects/test-liferay-workspace/modules/apps/sample-portlet/src/testFunctional/functions/AssertClick.function

```diff
@@ -0,0 +1,19 @@
+<definition default="assertClick">
```

Comment:
We can remove all of these testFunctional files, i think those are not releveant.